src/wayscript/secret_manager.py

```diff
@@ -0,0 +1,20 @@
+from urllib.error import HTTPError
+
+from . import utils, context, errors
+
+def set_secret(secret_key: str, secret_val: str) -> bool:
```

Comment:
You might want to run flake8 on the files changed.
```suggestion


def set_secret(secret_key: str, secret_val: str) -> bool:
```

Reply:
Ah, yeah, there's no lint settings file in the repo so it wasn't enabled. I'll take a linting pass